### ISSUE TITLE
Enhancement: Allow inclusion of multiple configuration files

### ISSUE THREAD

**Issue by nigelflack** [realm/SwiftLint#1352]
I'd like to propose adding a mechanism for including modular configuration files. Also requires the ability to name them other than .swiftlint.yml

i.e.:

```
disabled_rules:
  - trailing_whitespace
  - trailing_newline

[...stuff...]

include_subconfig:
  - customconfig1.yml                # some modular configuration (i.e. custom rules)
  - customconfig2.yml                # another modular configuration
```

I'm not sure the best way raise an enhancement request so opened an issue to gauge interest.

**Comment by acecilia** [comment]
Interested! :)

**Comment by fredpi** [comment]
There's still an active PR addressing this: https://github.com/realm/SwiftLint/pull/3058

**Comment by fredpi** [comment]
This is now implemented in #3058 and will be available in the next release!
